includes/class-plugin-manager.php

```diff
@@ -23,11 +23,18 @@ class Plugin_Manager {
 	 *
 	 * @return array of plugins info.
 	 */
-
 	public static function get_managed_plugins() {
 		$managed_plugins = [
 			'newspack-blocks'               => [
 				'Name'        => esc_html__( 'Newspack Blocks', 'newspack' ),
+				'Description' => esc_html__( 'Google Ad Manager integration.' ),
```

Comment:
Looks like the descriptions have ended up the wrong way round here :)

Reply:
Good catch: https://github.com/Automattic/newspack-plugin/pull/184/commits/d6803795783ee4a155ec303b1055d70d58a19b2b